code/game/gamemodes/eldritch_cult/eldritch_cult.dm

```diff
@@ -0,0 +1,69 @@
+/datum/game_mode/ecult
+	name = "eldritch cultist"
+	config_tag = "eldritch cultist"
+	report_type = "eldritch cultist"
+	antag_flag = ROLE_ECULT
+	false_report_weight = 5
+	protected_jobs = list("Prisoner","Security Officer", "Warden", "Detective", "Head of Security", "Captain")
+	restricted_jobs = list("AI", "Cyborg","Chaplain")
+	required_players = 0
+	required_enemies = 1
+	recommended_enemies = 4
+	reroll_friendly = 1
+	enemy_minimum_age = 0
+
+	announce_span = "danger"
+	announce_text = "Eldritch cultist has been spotted on the station!\n\
+	<span class='danger'>Cultist</span>: Accomplish your objectives.\n\
+	<span class='notice'>Crew</span>: Do not let the madman succeed!"
+
+	var/ecult_possible = 4 //hard limit on culties if scaling is turned off
+	var/num_ecult = 1
+	var/list/culties = list()
+
+/datum/game_mode/ecult/pre_setup()
+
+	if(CONFIG_GET(flag/protect_roles_from_antagonist))
+		restricted_jobs += protected_jobs
+
+	if(CONFIG_GET(flag/protect_assistant_from_antagonist))
+		restricted_jobs += "Assistant"
+
+	var/esc = CONFIG_GET(number/ecult_scaling_coeff)
+	if(esc)
+		num_ecult = max(1, min(round(num_players() / (esc * 2)) + 2, round(num_players() / esc)))
```

Comment:
so if the number of eldritch cultists can be greater than 1, why do all of the announcements and stuff assume that there's only one eldritch cultist?

Reply:
ah good catch